cmd/root.go

```diff
@@ -66,12 +66,23 @@ var rootCmd = &cobra.Command{
 
 		createLogger()
 
+		// set up the global viper config with values from
```

Comment:
`set up the global viper config with default values from ...`

Reply:
done